vault/utils.go

```diff
@@ -56,6 +56,33 @@ func replaceDoubleSlash(url string) string {
 	return strings.ReplaceAll(url, "//", "/")
 }
 
+// Util func to obtain filtered mounts from all mounts
+func filterMounts(in map[string]*api.MountOutput, mountType string) map[string]*api.MountOutput {
+	filtered := map[string]*api.MountOutput{}
+
+	for i, mount := range in {
```

Comment:
i here is the mount path right? when i see "i" i assume index, maybe make it p or path?

Reply:
Also, since you've written a nice filter function anyway, see if htere's any other logic that should be using this instead?
"table_kv_secret" is one that could use this.

Reply:
Sure, makes sense.